src/main/java/net/aboodyy/localtime/DateManager.java

```diff
@@ -20,28 +20,37 @@
 
 package net.aboodyy.localtime;
 
-import me.clip.placeholderapi.PlaceholderAPIPlugin;
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
-import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
-import org.bukkit.event.player.PlayerQuitEvent;
-import org.bukkit.scheduler.BukkitRunnable;
 
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
 import java.net.InetSocketAddress;
 import java.net.URL;
 import java.net.URLConnection;
 import java.text.SimpleDateFormat;
-import java.util.*;
+import java.util.Date;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.UUID;
+import java.util.concurrent.*;
 
 public class DateManager implements Listener {
 
     private final Map<UUID, String> timezones;
-
-    DateManager() {
-        timezones = new HashMap<>();
+    private final Map<String, String> cache;
```

Comment:
Thank you for pointing that out. Previously I was using my own system to avoid additional dependencies, and have more granular control over the cache, however in this context that really isn't necessary. Latest commits now reflect the appropriate changes